### ISSUE TITLE
G7 - Markup Validation issue on global search bar

### ISSUE THREAD

**Issue by jpinsent** [ccswbs/hjckrrh#312]
https://validator.w3.org/nu/?doc=https%3A%2F%2Faoda-web.uoguelph.ca%2Faodademo%2Fnews

```
Error: Bad value form for attribute role on element form.
From line 90, column 9; to line 90, column 161
↩↩        <form class="navbar-form navbar-right" role="form" id="searchbox_011117603928904778939:tp3ks5ha2dw" name="searchform" action="//www.uoguelph.ca/search/">↩     
```

**Comment by brharp** [comment]
A few more:

``````
Warning: Element nav does not need a role attribute.

From line 85, column 3; to line 85, column 73

bar -->↩ <nav class="navbar navbar-default navbar-static-top" role="navigation">↩ <

Warning: Element form does not need a role attribute.

From line 105, column 9; to line 105, column 161

↩↩ <form class="navbar-form navbar-right" role="form" id="searchbox_011117603928904778939:tp3ks5ha2dw" name="searchform" action="//www.uoguelph.ca/search/">↩

Warning: The banner role is unnecessary for element header.

From line 220, column 3; to line 220, column 84

</div>↩ <header id="navbar" role="banner" class="navbar navbar-static-top navbar-inverse">↩ <di

Warning: Element nav does not need a role attribute.

From line 236, column 11; to line 236, column 33

<nav role="navigation">↩ 
``````

**Comment by mmafe** [comment]
Note - it might be worth leaving the navigation role on nav for the sake of older screen readers. See https://www.w3.org/WAI/GL/wiki/Using_HTML5_nav_element

**Comment by abanuog** [comment]
Decided to leave as is.

**Comment by abanuog** [comment]
addressing the errors in #482 and #237

**Comment by mmafe** [comment]
Fixed by #862
